src/components/BooksList.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes, { Array } from 'prop-types';
+import Book from './Book';
+
+// eslint-disable-next-line react/prop-types
+const BookList = ({ books }) => (
+  <table>
+    <thead>
+      <tr>
+        <th>Book ID</th>
+        <th>Title</th>
+        <th>Category</th>
```

Comment:
- [ ] We need to move BookForm from `components` directory to the `containers` directory.